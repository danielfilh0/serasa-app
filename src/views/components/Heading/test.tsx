import { render } from '@testing-library/react'

import { Heading } from '.'

describe('<Heading />', () => {
  it('should render h1 as default', () => {
    const { container } = render(<Heading>Heading</Heading>)

    expect(container.querySelector('h1')).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })

  it('should render other tag', () => {
    const { container } = render(<Heading variant="h3">Heading</Heading>)

    expect(container.querySelector('h3')).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
