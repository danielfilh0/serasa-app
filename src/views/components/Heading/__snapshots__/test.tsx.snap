// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> should render h1 as default 1`] = `
<h1
  class="text-[2.5rem] md:text-[3rem] leading-[44px] md:leading-[52px] tracking-[-1.2px] font-bold text-dark-high"
>
  Heading
</h1>
`;

exports[`<Heading /> should render other tag 1`] = `
<h3
  class="text-[2.5rem] md:text-[3rem] leading-[44px] md:leading-[52px] tracking-[-1.2px] font-bold text-dark-high"
>
  Heading
</h3>
`;
